model.js

```diff
@@ -11,21 +11,34 @@ const fetchAllTopics = () => {
 
 };
 
-const fetchAllArticles = () => {
-  const sqlQuery = `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count
+const fetchAllArticles = (sort_by = `created_at`, order = `DESC`, topic) => {
+  const acceptedOrders = ['ASC', 'DESC']
```

Comment:
I like this a lot, though you'd probably also benefit from accepted sort_bys and, slightly differently, finding a way to see if the topic requested exists or not (hint: you can utilise your GET topics endpoint to do so).